### PR TITLE
:goal_net: Setup the user command line error handling

### PR DESCRIPTION
**Pull request by Thomaltarix**
I added the command-line error handling.

Now, the tree arguments are handled and stored in a class.
There's getters to access those variables.

For more details, check this issue: #6

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,7 +23,7 @@ NAME		=	plazza
 
 WARNINGS	=	-Wall -Wextra -Wshadow
 
-INCLUDE		=	-I./include
+INCLUDE		=	-I./src/Errors
```
Comment (Peralban) [inline Makefile:25]: ???????????????????????????????

**Review by Peralban — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (Thomaltarix) [reply Makefile]: Done

File: src/Arguments.cpp [hunk 2]
```diff
@@ -0,0 +1,51 @@
+/*
+** EPITECH PROJECT, 2024
+** Plazza
+** File description:
+** Arguments
+*/
+
+#include "Arguments.hpp"
+
+Plazza::Arguments::Arguments(int ac, char **av)
```
[the remark this answers is not in the document]
Comment (Thomaltarix) [reply src/Arguments.cpp:10]: Done

File: src/Arguments.cpp [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+            displayUsage();
+            throw Errors::WrongCookNumber("Invalid number of cooks");
+        } if (std::stof(av[3]) <= 0) {
+            displayUsage();
+            throw Errors::WrongRestockTime("Invalid restock time");
+        }
+    } catch (Errors::ArgumentsErrors &e) {
+        std::cerr << e.what() << std::endl;
+        exit(84);
+    }
```
Comment (landryarki) [inline src/Arguments.cpp:29]: maybe consider putting displayUsage in the catch ??

Comment (Thomaltarix) [reply src/Arguments.cpp:25]: Done

Comment (landryarki) [reply src/Arguments.cpp:25]: you still need to catch errors of std::stoi std::stof

**Review by landryarki — CHANGES_REQUESTED** [review]

File: src/Arguments.cpp (hunk 3 above)
Comment (Thomaltarix) [reply src/Arguments.cpp:24]: Done

**Review by landryarki — APPROVED** [review]
:prince: nice

**Pull request merged**